hadoop-project/pom.xml

```diff
@@ -114,8 +114,8 @@
     <bouncycastle.version>1.78.1</bouncycastle.version>
 
     <!-- Required for testing LDAP integration -->
-    <apacheds.version>2.0.0.AM26</apacheds.version>
-    <ldap-api.version>2.0.0</ldap-api.version>
+    <apacheds.version>2.0.0.AM27</apacheds.version>
+    <ldap-api.version>2.1.5</ldap-api.version>
```

Comment:
The `ldap-api.version` can be upgraded to 2.1.7. I have tested it locally.

Reply:
Done.
2.0.0.AM27 ships with 2.1.5, but it's 1.5 years old at this point, so it makes sense that some dependencies have newer verions.